cs/ccxt/base/Exchange.Options.cs

```diff
@@ -80,6 +80,7 @@ public partial class Exchange
     public string password { get; set; }
     public string uid { get; set; }
     public string accountId { get; set; }
+    public int minFundingAddressLength { get; set; }
```

Comment:
you declare it here but it's never initialized ?

Reply:
ah, forgot to set to 1 (like in other langs https://github.com/ccxt/ccxt/blob/master/php/Exchange.php#L266 )

Reply:
done.